### DISCUSSION TITLE
How do I flip the model 360 degrees up and down using <model-viewer> in HTML ？

### DISCUSSION THREAD

**Discussion by xdp6688** [google/model-viewer#3374] [Ideas]
How do I flip the model 360 degrees up and down using <model-viewer> in HTML ？
thank you!

**Comment by daniel-packard** [comment]
Hey - you are not the first to ask

- https://github.com/google/model-viewer/discussions/3314
- https://github.com/google/model-viewer/discussions/3219

Long story short - it is not currently possible to exceed this limit, but the maintainer is open to a pull request.
